packages/gallery/tests/drivers/reactDriver.js

```diff
@@ -178,6 +178,10 @@ class galleryDriver {
     };
   }
 
+  getContainer() {
+    return this.find.selector('#pro-gallery-container-default-dom-id');
```

Comment:
@yuvalab 
What will happen here when the someone in the test will define an ID as a prop ?

Reply:
you mean i need to let props.id  || the default string?

Reply:
Yes

Reply:
If the user decides in his test that he wants to use an `id` in props - the `getContainer` breaks